types/tar/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "private": true,
     "dependencies": {
-        "minipass": "^4.0.0"
+        "minipass": "^5.0.0"
```

Comment:
I compared the types in https://unpkg.com/browse/minipass@5.0.0/index.d.ts  to https://unpkg.com/browse/minipass@4.2.8/index.d.ts, and they are different. I guess we should stick with 4 for now since 4.2.8 just published and fixes the compile error.

(And I suspect that 5.0's types are still wrong.)